### PR TITLE
76836 KS5 school AcademicProgress and AppliedGeneralProgress

### PR DESCRIPTION
**Pull request by CathLass**

### CODE REVIEW

File: ApplyToBecomeInternal/ApplyToBecomeInternal/ViewModels/KeyStage5PerformanceTableViewModel.cs [hunk 1]
```diff
@@ -17,18 +17,19 @@ public class KeyStage5PerformanceTableViewModel
 		
 		public static KeyStage5PerformanceTableViewModel Build(KeyStage5PerformanceResponse keyStage5Performance)
 		{
-			return new KeyStage5PerformanceTableViewModel
+			var temp = new KeyStage5PerformanceTableViewModel
```
Comment (danielburnley) [inline ApplyToBecomeInternal/ApplyToBecomeInternal/ViewModels/KeyStage5PerformanceTableViewModel.cs:20]:
```suggestion
			return new KeyStage5PerformanceTableViewModel

```

File: ApplyToBecomeInternal/ApplyToBecomeInternal/ViewModels/KeyStage5PerformanceTableViewModel.cs [hunk 2]
```diff
@@ -17,18 +17,19 @@ public class KeyStage5PerformanceTableViewModel
...
 				AcademicAverage = keyStage5Performance.AcademicQualificationAverage.FormatValue(),
-				AppliedGeneralProgress = ((decimal?)null).FormatValue(),
+				AppliedGeneralProgress = keyStage5Performance.AppliedGeneralProgress.NotDisadvantaged.FormatValue(),
 				AppliedGeneralAverage = keyStage5Performance.AppliedGeneralQualificationAverage.FormatValue(),
 				NationalAverageAcademicProgress = ((decimal?)null).FormatValue(),
 				NationalAverageAcademicAverage = keyStage5Performance.NationalAcademicQualificationAverage.FormatValue(),
 				NationalAverageAppliedGeneralProgress = ((decimal?)null).FormatValue(),
 				NationalAverageAppliedGeneralAverage = keyStage5Performance.NationalAppliedGeneralQualificationAverage.FormatValue()
 			};
+			return temp;
```
Comment (danielburnley) [inline ApplyToBecomeInternal/ApplyToBecomeInternal/ViewModels/KeyStage5PerformanceTableViewModel.cs:32]:
```suggestion

```

**Review by danielburnley — COMMENTED** [review]
Can probably change this back to how it was?

File: ApplyToBecomeInternal/ApplyToBecomeInternal/ViewModels/KeyStage5PerformanceTableViewModel.cs (hunk 2 above)
Comment (CathLass) [reply ApplyToBecomeInternal/ApplyToBecomeInternal/ViewModels/KeyStage5PerformanceTableViewModel.cs:32]: thanks. done

**Review by danielburnley — APPROVED** [review]

**Pull request merged**